### PR TITLE
[JBPM-10020]-Add incrementing and decrementing prometheus metric for …

### PR DESCRIPTION
**Pull request by VaniHaripriya**
JIRA- https://issues.redhat.com/browse/JBPM-10020

Created a new command to throw execution error to use it for testing a new prometheus metric created for kie server execution errors.

Related Pull Request: https://github.com/kiegroup/droolsjbpm-integration/pull/2716

### CODE REVIEW

File: jbpm-services/jbpm-executor/src/main/java/org/jbpm/executor/commands/JobExecutionErrorCommand.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    
+    private static final Logger logger = LoggerFactory.getLogger(JobExecutionErrorCommand.class);
+
+    @SuppressWarnings("null")
+	public ExecutionResults execute(CommandContext ctx) {    	
+    	ctx = null; //Assigning null value to throw null pointer exception
+		try {
+			logger.info("Command executed on executor with data {}", ctx.getData());
+		} catch (NullPointerException e) {
+
```
Comment (elguardian) [inline jbpm-services/jbpm-executor/src/main/java/org/jbpm/executor/commands/JobExecutionErrorCommand.java:40]: this won't add up as an error as you are catching the null pointer. to simulate an error you just can
throw new RuntimeException()

File: jbpm-services/jbpm-executor/src/main/java/org/jbpm/executor/commands/JobExecutionErrorCommand.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+ * Just for testing purpose.
+ * 
+ */
+public class JobExecutionErrorCommand implements Command{
+    
+    private static final Logger logger = LoggerFactory.getLogger(JobExecutionErrorCommand.class);
+
+    @SuppressWarnings("null")
+	public ExecutionResults execute(CommandContext ctx) {    	
+    	ctx = null; //Assigning null value to throw null pointer exception
```
Comment (gmunozfe) [inline jbpm-services/jbpm-executor/src/main/java/org/jbpm/executor/commands/JobExecutionErrorCommand.java:36]: this assignment is not correct, better follow Enrique's approach.

**Review by gmunozfe — CHANGES_REQUESTED** [review]

File: jbpm-services/jbpm-executor/src/main/java/org/jbpm/executor/commands/JobExecutionErrorCommand.java [hunk 3]
```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2017 Red Hat, Inc. and/or its affiliates.
```
Comment (gmunozfe) [inline jbpm-services/jbpm-executor/src/main/java/org/jbpm/executor/commands/JobExecutionErrorCommand.java:2]:
```suggestion
 * Copyright 2022 Red Hat, Inc. and/or its affiliates.
```

**Comment by VaniHaripriya** [comment]
> If this is for testing, shouldn't it be at "test" folder instead of "main"?

Agree with you @gmunozfe , will move it to test folder.

**Review by elguardian — APPROVED** [review]

**Comment by gmunozfe** [comment]
jenkins retest this

**Comment by VaniHaripriya** [comment]
> > If this is for testing, shouldn't it be at "test" folder instead of "main"?
> 
> Agree with you @gmunozfe , will move it to test folder.

@gmunozfe ,Prometheus Integration test can not access the class in test folder, as we are accessing jbpm-executor from drools-jbpm-integration through the dependency jar. So, leaving JobExecutionErrorCommand.java in main folder.

**Pull request merged**